src/metabase/lib/drill_thru/quick_filter.cljc

```diff
@@ -53,7 +53,15 @@
    [metabase.lib.temporal-bucket :as lib.temporal-bucket]
    [metabase.lib.types.isa :as lib.types.isa]
    [metabase.lib.underlying :as lib.underlying]
-   [metabase.util.malli :as mu]))
+   [metabase.util.malli :as mu]
+   [metabase.util.number :as u.number]))
+
+(defn- maybe-bigint->value-clause
+  [value]
+  (let [number (when (string? value) (u.number/parse-bigint value))]
+    (if (some? number)
+      (lib.options/ensure-uuid [:value {:base-type :type/BigInteger :effective-type :type/BigInteger} (str number)])
+      value)))
```

Comment:
`if-let` is handy for this sort of thing. Also only `false` and `nil` are falsy in Clojure(Script), so there's no need to be cautious of 0 (or `""`) with `some?`.
```suggestion
  (if-let [number (when (string? value) (u.number/parse-bigint value))]
    (lib.options/ensure-uuid [:value {:base-type :type/BigInteger :effective-type :type/BigInteger} (str number)])
    value))
```